pkgs/development/python-modules/minimal-snowplow-tracker/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "minimal-snowplow-tracker";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-rKv3Vy2w5/XL9pg9SV7vVAgfcb45IzDrOq25zLOdqqQ=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    six
+  ];
+
+  pythonImportsCheck = [
+    "snowplow_tracker"
+  ];
+
+  meta = with lib; {
+    description = "Minimal snowplow event tracker";
+    homepage = "https://pypi.org/project/minimal-snowplow-tracker";
```

Comment:
Do we have the source code somewhere?

Reply:
I found a repository for it and updated the link.